configs/SEQUREH7V2/config.h

```diff
@@ -111,15 +133,22 @@
 
 #define MAG_I2C_INSTANCE (I2CDEV_1)
 #define BARO_I2C_INSTANCE (I2CDEV_1)
+
 #define ADC_INSTANCE ADC1
+
 #define DEFAULT_BLACKBOX_DEVICE BLACKBOX_DEVICE_FLASH
-#define DEFAULT_DSHOT_BURST DSHOT_DMAR_OFF
+
 #define DEFAULT_CURRENT_METER_SOURCE CURRENT_METER_ADC
+#define DEFAULT_CURRENT_METER_SCALE 1052
+
 #define DEFAULT_VOLTAGE_METER_SOURCE VOLTAGE_METER_ADC
 #define DEFAULT_VOLTAGE_METER_SCALE 110
-#define DEFAULT_CURRENT_METER_SCALE 1052
+
 #define BEEPER_INVERTED
+
 #define MAX7456_SPI_INSTANCE SPI1
+
 #define GYRO_1_SPI_INSTANCE SPI2
-#define FLASH_SPI_INSTANCE SPI3
 #define GYRO_1_ALIGN CW90_DEG
+
+#define FLASH_SPI_INSTANCE SPI3
```

Comment:
- No new line on EOF
```suggestion
#define FLASH_SPI_INSTANCE SPI3

```